### PR TITLE
Removed pel id and replaced with event id

### PR DESCRIPTION
**Pull request by Renuka9527**
Defect: https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=576103
Description: On the hardware deconfiguration page of the BMC GUI the PEL ID column is renamed or "Event ID" since that is what is listed in that column.

### CODE REVIEW

File: src/views/Settings/HardwareDeconfiguration/MemoryDimms.vue [hunk 1]
```diff
@@ -151,10 +151,10 @@ export default {
           tdClass: 'text-nowrap',
         },
         {
-          key: 'pelID',
+          key: 'eventID',
```
Comment (Nikhil-Ashoka) [inline src/views/Settings/HardwareDeconfiguration/MemoryDimms.vue:154]: To use this as a key, we need to rename or create eventID in the store.

File: src/views/Logs/DeconfigurationRecords/DeconfigurationRecords.vue [hunk 2]
```diff
@@ -272,8 +272,8 @@ export default {
           sortable: true,
         },
         {
-          key: 'pelID',
-          label: this.$t('pageDeconfigurationRecords.table.pelId'),
+          key: 'EventID',
```
Comment (Nikhil-Ashoka) [inline src/views/Logs/DeconfigurationRecords/DeconfigurationRecords.vue]: Keep this 'E' in lower case, Both here and also in store. As it is variable.

**Review by Nikhil-Ashoka — CHANGES_REQUESTED** [review]

File: src/store/modules/Logs/DeconfigurationRecordsStore.js [hunk 3]
```diff
@@ -82,7 +82,7 @@ const DeconfigurationRecordsStore = {
                 uri: log['@odata.id'],
                 severity: Severity,
                 location: LocationCode,
-                pelID: pelId,
+                EventID: eventId,
```
Comment (Nikhil-Ashoka) [inline src/store/modules/Logs/DeconfigurationRecordsStore.js]: Lowercase e is preferred in the left hand side: EventID.

File: src/views/Settings/HardwareDeconfiguration/MemoryDimms.vue (hunk 1 above)
Comment (Renuka9527) [reply src/views/Settings/HardwareDeconfiguration/MemoryDimms.vue:154]: Addressed

File: src/store/modules/Logs/DeconfigurationRecordsStore.js (hunk 3 above)
Comment (Renuka9527) [reply src/store/modules/Logs/DeconfigurationRecordsStore.js]: Addressed

File: src/views/Logs/DeconfigurationRecords/DeconfigurationRecords.vue (hunk 2 above)
Comment (Renuka9527) [reply src/views/Logs/DeconfigurationRecords/DeconfigurationRecords.vue]: Addressed

**Review by Nikhil-Ashoka — APPROVED** [review]

**Pull request merged**